ci-templates/.github/workflows/deploy-to-pantheon-live.yml

```diff
@@ -53,30 +53,22 @@ jobs:
           pantheon: 'true'
           ssh-key: ${{ secrets.ALLEY_CI_PANTHEON_SSH_KEY }}
 
-  deploy-to-pantheon:
-    runs-on: ubuntu-latest
-    needs: build-and-sync
-    steps:
-      - name: Install PHP
-        run: sudo apt-get install php
-
       - name: Install Terminus
         run: |
           mkdir -p /tmp/terminus && cd /tmp/terminus
           curl -L https://github.com/pantheon-systems/terminus/releases/download/${{ env.TERMINUS_VERSION }}/terminus.phar --output terminus
           chmod +x terminus
-          ./terminus self:update
           sudo mv /tmp/terminus/terminus /usr/local/bin/terminus
 
       - name: Authenticate with Terminus
         run: terminus auth:login --machine-token=${{ secrets.PANTHEON_MACHINE_TOKEN }}
 
       - name: Deploy to Test
         run: |
-          terminus env:deploy ${{ env.PANTHEON_SITE }}.test --note="Deploying ${{ github.event.head_commit.message }} from GitHub Actions"
+          terminus env:deploy ${{ env.PANTHEON_SITE }}.test --note="$(git log -1 --pretty=%B)"
```

Comment:
what about 

```
commit_message="$( git log -1 --pretty=%B | sed 's/"/\\"/g' )"
    terminus env:deploy ${{ env.PANTHEON_SITE }}.test --note="Deploying $commit_message from GitHub Actions"
    ```

Reply:
just be explicit about escaping the doublequotes? Still retain the 'fluff' around the commit message?

Reply:
That commit message could be multiple lines and wrapping it with quotes would look a little off. I think prefixing it with `Deploying from GitHub Actions:` could be a good compromise.